### ISSUE TITLE
Multiple file upload allowed file types field does not have form label

### ISSUE THREAD

**Issue by hashraf88** [Alfresco/alfresco-ng2-components#379]
Form label required on field

![screen shot 2016-07-07 at 07 24 45](https://cloud.githubusercontent.com/assets/13200338/16644307/e8087d3e-4413-11e6-88a2-b973c8491eac.png)
